tests/api/test_projects.py

```diff
@@ -291,8 +288,22 @@ def test_invalid_id(self, flask_api_url):
         print(response.text)
         assert response.status_code == 404
 
-    def test_success_getall(self, flask_api_url):
-        assert True
+    def test_success_getall(self, flask_api_url, manifest_validator,
+                            mongo_client, enter_data_using_post):
+        response = requests.get(flask_api_url + "/api/projects")
+        print(response.status_code)
+        assert response.status_code == 200
+        projects = response.json()
```

Comment:
So then there are no projects in here, right? Whats the purpose of this test then.